Add hardfork meta after Istanbul (Berlin) (#2070)
ethereum/EIPs: EIPS/eip-2070.md

---
eip: 2070

> title: "Hardfork Meta: Berlin"

Reviewer: ```suggestion
title: Network Upgrade Meta: <April 2020/To Be Named Later>
```

author: Alex Beregszaszi (@axic)
discussions-to: <url>
type: Meta
status: Draft
created: 2019-05-20
requires: 1679
---

## Abstract

> This meta-EIP specifies the changes included in the Ethereum hardfork named Berlin.

Reviewer: ```suggestion
This meta-EIP specifies the changes included in the April 2020 Roadmap Network Upgrade &lt;to be named later&gt;
```

## Specification

> - Codename: Berlin

Reviewer: ```suggestion
- Codename: &lt;To be named later&gt;
```

> - Activation: TBD

Reviewer: ```suggestion
- Activation: 15 April 2020 &lt;tentative&gt;
```

### Included EIPs

- TBD

### Accepted EIPs

- TBD

### Proposed EIPs

- TBD

## Timeline

> * 2019-09-20 (Fri) hard deadline to accept proposals for "Berlin"

Reviewer: From the cheesy schedule we can actually project more dates.  And Align them to AllCoreDevs calls (assuming 2 week cadence)
```suggestion
* 2019-07-19 (Fri) All Core Devs "Kickoff"
* 2019-09-27 (Fri) Hard Deadline for Proposals
* 2019-11-22 (Fri) Hard Deadline for EIP Acceptance
* 2020-01-17 (Fri) Client Implementation Soft Deadline
* 2020-02-19 (Wed) Testnet Network Upgrade Target
* 2020-04-22 (Wed) Mainnet Network Upgrade Target
```
Also, it would be cool in the ACD call if the "Kickoff" consisted of an EIP editor pressing the "merge" button.
